cmd/cli/config/config.go

```diff
@@ -24,21 +24,20 @@ type Config interface {
 
 // cliConfig implements Config interface
 type cliConfig struct {
-	OutFormat  string `required:"false" default:"" yaml:"output_format"`
-	Keystore   string `required:"false" default:"" yaml:"key_store"`
-	Passphrase string `required:"false" default:"" yaml:"pass_phrase"`
+	Eth       accounts.EthConfig `yaml:"ethereum"`
+	OutFormat string             `required:"false" default:"" yaml:"output_format"`
 }
 
 func (cc *cliConfig) OutputFormat() string {
 	return cc.OutFormat
 }
 
 func (cc *cliConfig) PassPhrase() string {
-	return cc.Passphrase
```

Comment:
I'd better use Eth directly or via method (e.g. Eth().Passphrase()), but ok for now